injection.js

```diff
@@ -30,6 +30,7 @@ function init(){
 		}
 		replaceElements();
 		linkNPCs();
+		initMap();
```

Comment:
Why are we running initMap in injection.js?
Are we also going to have the map on the guide page as well?